geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -155,11 +177,10 @@
           + ";org.apache.geode.internal.cache.tier.sockets.VersionedObjectList" // putAll
           + ";org.apache.shiro.*;org.apache.shiro.authz.*;org.apache.shiro.authc.*" // security
                                                                                     // services
-          + ";org.apache.logging.log4j.**" // export logs
-          + ";org.apache.geode.connectors.jdbc.internal.**"
+          + ";org.apache.logging.log4j.Level" + ";com.healthmarketscience.rmiio.RemoteInputStream"
```

Comment:
Would you mind leaving the comments on where/what caused these to be added.

Reply:
I've put the comments back in place